draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
Considering that there is a strong mandate to keep protocols secure and that the required feature (early media) can be implemented without jeopardizing security I would prefer not allow for the handshake to continue until it can be verified, i.e. until fingerprints and tls-id are received.